js/.eslintrc.cjs

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
I've used [eslint settings from doc-builder](https://github.com/huggingface/doc-builder/blob/main/kit/.eslintrc.cjs)

please let me know if I should use [moon-landing one instead](https://github.com/huggingface/moon-landing/blob/main/server/.eslintrc.js)